docker/manifest.tmpl

```diff
@@ -0,0 +1,25 @@
+image: harnesscommunitytest/archive-plugin:{{#if build.tag}}{{trimPrefix "v" build.tag}}{{else}}latest{{/if}}
```

Comment:
remove harnesscommunity references and replace it with `plugins`

Reply:
I need access to plugins.

Reply:
Changed it to plugins/archive